pom.xml

```diff
@@ -114,7 +114,7 @@
 			<dependency>
 				<groupId>life.qbic</groupId>
 				<artifactId>data-model-lib</artifactId>
-				<version>2.0.0</version>
+				<version>2.1.0-SNAPSHOT</version>
```

Comment:
Hm, can we make a release of the data model lib first please?

Reply:
So you mean to release the snapshot version from development?

Reply:
to release a stable version 2.1.0 with the current development state, yes-

Reply:
addressed :)